cmiles/generator.py

```diff
@@ -116,12 +118,14 @@ def to_molecule_id(molecule, canonicalization='openeye'):
         raise TypeError("canonicalization must be either 'openeye' or 'rdkit'")
 
     smiles['Standard_InChI'], smiles['InChIKey'] = to_inchi_and_key(molecule_copy)
```

Comment:
I missed this the first time, but shouldn't this be `standard_inchi/inchikey` to match all other snake case keys?

Reply:
Good point.